src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/FlowHsTopology.java

```diff
@@ -737,7 +738,12 @@ private void haFlowDeleteHub(TopologyBuilder topologyBuilder, PersistenceManager
     }
 
     private void haFlowReadBolt(TopologyBuilder topologyBuilder, PersistenceManager persistenceManager) {
-        HaFlowReadBolt bolt = new HaFlowReadBolt(persistenceManager);
+        HaFlowReadConfig config = HaFlowReadConfig.builder()
+                .readOperationRetriesLimit(topologyConfig.getYFlowReadRetriesLimit())
+                .readOperationRetryDelay(Duration.ofMillis(topologyConfig.getYFlowReadRetryDelayMillis()))
+                .lifeCycleEventComponent(ZooKeeperSpout.SPOUT_ID)
```

Comment:
As you are using a lifeCycle you need to add `HA_FLOW_READ_BOLT` into zookeeper liveCycle. It is related to blue green deployment. 

Try to look to YFLOW_READ_BOLT in this file and do the same for HA_FLOW_READ_BOLT

Reply:
Fixed